controller/network/smart.go

```diff
@@ -24,13 +24,17 @@ import (
 
 func (network *Network) smart() {
 	log := pfxlog.Logger()
-	log.Debugf("smart network processing")
+	log.Trace("smart network processing")
 
 	/*
 	 * Order sessions in decreasing overall latency order
 	 */
 	sessions := network.GetAllSessions()
-	log.Debugf("observing [%d] sessions", len(sessions))
+	if len(sessions) > 0 {
+		log.Debugf("observing [%d] sessions", len(sessions))
+	} else {
+		log.Tracef("observing [%d] sessions", len(sessions))
+	}
```

Comment:
Why the conditional `Debugf`/`Tracef`?

Reply:
just to reduce the log noise when you're testing data-flow bits with debug on a system